Listenbrainz/UI/Components/TabBarAccessor.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  TabBarAccessor.swift
+//  Listenbrainz
+//
+//  Created by Jasjeet Singh on 12/04/25.
+//
+
+import SwiftUI
+import UIKit
+
+class InsetsHolder: ObservableObject {
+    @Published var tabBarHeight: CGFloat = 0
+}
+
+struct TabBarAccessor: UIViewControllerRepresentable {
```

Comment:
This is an overcomplicated logic for just managing the padding. I have never handled things this way anywhere. What inspired you to make a custom UIKit controller for this? We should handle things just the SwiftUI way.

Reply:
This is the only way we can get height of the Tab without hardcoding things. Its not overcomplicated, the complications come just because its implementing that protocol.